ui/shared/IconButton.qml

```diff
@@ -4,26 +4,33 @@ import QtQuick.Layouts 1.3
 import Qt.labs.platform 1.1
 import "../imports"
 
-Rectangle {
-    signal clicked
+RoundButton {
     property int iconWidth: 14
     property int iconHeight: 14
-    property alias icon: imgIcon
+    property alias iconImg: imgIcon
     property bool clickable: true
     property string iconName: "plusSign"
+    property color color: Style.current.blue
+
+    icon.width: iconWidth
+    icon.height: iconHeight
```

Comment:
This is so we can still use the normal `icon` property? If so, those two lines are not 100% necessary, because someone using `IconButton` could just modify `icon.width` themselves.

Reply:
Yea so this is so that existing usages of this component don't break if they've used `iconWidth` and `iconHeight`. We just delegate it to `icon.width` and `icon.height`, mostly because `RoundedButton` already comes with an `icon` property. We, ourselves, don't really make use of `RoundedButton.icon`, it's more to make things work "as expected".

I'd say that we'll prolly revisit that in the future and introduce a simpler "Icon" component that is not a button, where don't run into those conflicts.

Reply:
Makes sense